content/src/view.js

```diff
@@ -1332,8 +1347,14 @@ $(window).on('resize.listing', function() {
 });
 
 function updatePaneLinks(pane) {
+  var DEFAULT_THUMBS = {
+    user: '//' + pencilcode.domain + '/image/user-128.png',
```

Comment:
For the user listing, can we omit the thumbnails and use a dense text-only layout?  It's not used very often, but it's sometimes used by teachers to find their students oddly-spelled usernames - and high density is more helpful than icons there.